### PR TITLE
core: display coin name when signing message

### LINKED ISSUE #1159: Do we print coin name during message signing?
We should print the Coin name during message signing so the attacker can not trick the user into signing a message with a different coin than intended.

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #1159. Also displays the coin shortname when showing address during the `verify_message` flow.

[UI diff](https://satoshilabs.gitlab.io/-/trezor/trezor-firmware/-/jobs/681119460/artifacts/master_diff/index.html)

### CODE REVIEW

File: core/src/apps/common/signverify.py [hunk 1]
```diff
@@ -41,21 +41,26 @@ def split_message(message: bytes) -> List[str]:
...
 
 
 async def require_confirm_verify_message(
-    ctx: wire.Context, address: str, header: str, message: bytes
+    ctx: wire.Context, address: str, coin: str, message: bytes
 ) -> None:
     text = Text("Confirm address", new_lines=False)
+    text.bold("Coin: {}".format(coin))
+    text.br()
+    text.br_half()
```
Comment (tsusanka) [inline core/src/apps/common/signverify.py:59]: I am thinking "Confirm address" is very confusing. Maybe we could also have "Verify {} message" as the header and then text.bold("Confirm address:")?

Comment (tsusanka) [reply core/src/apps/common/signverify.py:59]: So it will look like this:
```
Verify ETH message

Confirm address:
ABCDEF
```

Comment (mmilata) [reply core/src/apps/common/signverify.py]: Yeah that looks better - fixed.

**Comment by mmilata** [comment]
Rebased.

**Pull request merged**